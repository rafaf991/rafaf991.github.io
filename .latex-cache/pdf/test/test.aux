\relax 
\providecommand\hyper@newdestlabel[2]{}
\providecommand\HyField@AuxAddToFields[1]{}
\providecommand\HyField@AuxAddToCoFields[2]{}
\providecommand\babel@aux[2]{}
\@nameuse{bbl@beforestart}
\babel@aux{english}{}
\babel@aux{english}{}
\@writefile{toc}{\contentsline {section}{\numberline {1}Test1}{1}{section.1}\protected@file@percent }
\@writefile{toc}{\contentsline {subsection}{\numberline {1.1}test2}{1}{subsection.1.1}\protected@file@percent }
\gdef \@abspage@last{1}
