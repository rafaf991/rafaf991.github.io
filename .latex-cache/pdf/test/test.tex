


\documentclass{article}

\usepackage[utf8x]{inputenc}

\usepackage[T1]{fontenc}
\usepackage{listings}
\usepackage{physics}
\usepackage{amsthm}
\usepackage{amssymb}

\newtheorem{theorem}{Theorem}[section]
\newtheorem{example}[theorem]{Example}
\newtheorem{remark}[theorem]{Remark}



\newtheorem{proposition}{Proposition}
\newtheorem{corollary}{Corollary}[theorem]
\newtheorem{lemma}[theorem]{Lemma}
\newtheorem{definition}{Definition}


\newcommand{\CC}{\mathbb{C}}




\usepackage{tikz}

\usepackage{hyperref}


\usepackage[english]{babel}
\usepackage[english]{isodate}









\setcounter{footnote}{0}

\hypersetup{colorlinks=true,urlcolor=blue}


\title{test}


\author{Rafael C.}



\date{2000-01-28}



\begin{document}

\maketitle

\tableofcontents


\section{Test1}


This should be written in latex, and will not be put in the html version.

\subsection{test2}


We shall first prove the following.

Lemma (Eigenspace orthogonality)

For any symmetric matrix $M$, and any two eigenvalues $\lambda \ne \lambda'$, the eigenvectors for $\lambda$ are orthogonal to the eigenvectors for $\lambda'$.


It can then be used to prove the following.

Theorem (Diagonalisation)

Any symmetric matrix can be diagonalised in an orthogonal basis.


\begin{theorem}
Esta es una prueba  

$$\qty(\int_0 \mathbb{R}) $$
\end{theorem}

\begin{proof}
Hola
\end{proof}

\begin{itemize}
\item testa sdasd
\item sdadas
\item 3
\end{itemize}


\end{document}
