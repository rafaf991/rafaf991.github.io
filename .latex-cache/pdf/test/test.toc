\babel@toc {english}{}\relax 
\babel@toc {english}{}\relax 
\contentsline {section}{\numberline {1}Test1}{1}{section.1}%
\contentsline {subsection}{\numberline {1.1}test2}{1}{subsection.1.1}%
