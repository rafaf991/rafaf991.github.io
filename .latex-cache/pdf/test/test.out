\BOOKMARK [1][-]{section.1}{\376\377\000T\000e\000s\000t\0001}{}% 1
\BOOKMARK [2][-]{subsection.1.1}{\376\377\000t\000e\000s\000t\0002}{section.1}% 2
